works.html

```diff
@@ -71,7 +71,300 @@
+                      src="./assets/images/works/check-mark.svg"
+                      alt="mark"
+                    />
+                    <div class="main__wrapper-common-best-item-text">
+                      <span class="title-best">Мы лучшие в этом</span>
+                      <p class="text-best">
+                        Мы понимаем как супер тяжело делать помогаем новичкам
+                      </p>
+                    </div>
+                  </div>
+                  <div class="main__wrapper-common-best-item">
+                    <img
+                      class="main__wrapper-common-best-item-picture"
+                      src="./assets/images/works/check-mark.svg"
+                      alt="mark"
+                    />
+                    <div class="main__wrapper-common-best-item-text">
+                      <span class="title-best">Мы лучшие в этом</span>
+                      <p class="text-best">
+                        Мы понимаем как супер тяжело делать помогаем новичкам
+                      </p>
+                    </div>
+                  </div>
+                  <div class="main__wrapper-common-best-item">
+                    <img
+                      class="main__wrapper-common-best-item-picture"
+                      src="./assets/images/works/check-mark.svg"
+                      alt="mark"
+                    />
+                    <div class="main__wrapper-common-best-item-text">
+                      <span class="title-best">Мы лучшие в этом</span>
+                      <p class="text-best">
+                        Мы понимаем как супер тяжело делать помогаем новичкам
+                      </p>
+                    </div>
+                  </div>
+                  <div class="main__wrapper-common-best-item">
+                    <img
+                      class="main__wrapper-common-best-item-picture"
+                      src="./assets/images/works/check-mark.svg"
+                      alt="mark"
+                    />
+                    <div class="main__wrapper-common-best-item-text">
+                      <span class="title-best">Мы лучшие в этом</span>
+                      <p class="text-best">
+                        Мы понимаем как супер тяжело делать помогаем новичкам
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="main__wrapper-common-why-picture">
+                <img class="main__wrapper-common-why-pic"
+                  src="./assets/images/works/map-scheme.png"
+                  alt="map-scheme"
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="main__container-additional">
+          <div class="main__wrapper">
+            <h3 class="common-text-heading">Новости компании</h3>
+            <div class="main__wrapper-projecting">
+              <div class="main__wrapper-projecting-item">
```

Comment:
Вот этот элемент ну прям тоочно нужно было сделать как блок по БЭМ-у. Так как он может много где использоваться, а в твоём случае, так 3 раза.
```suggestion
              <div class="projecting-item">
```